src/test/kotlin/no/fdk/concept_catalog/contract/SearchConcepts.kt

```diff
@@ -1,109 +1,103 @@
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
 import org.springframework.http.HttpMethod
 import org.springframework.http.HttpStatus
-import org.springframework.test.context.ContextConfiguration
 import kotlin.test.assertEquals
 
-private val mapper = JacksonConfigurer().objectMapper()
-
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
-@SpringBootTest(
-    properties = ["spring.profiles.active=contract-test"],
-    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
-)
-@ContextConfiguration(initializers = [ApiTestContext.Initializer::class])
 @Tag("contract")
-class SearchConcepts : ApiTestContext() {
-
+class SearchConcepts : ContractTestsBase() {
     val sortByModified = SortField(field = SortFieldEnum.SIST_ENDRET, direction = SortDirection.DESC)
 
     @Test
     fun `Unauthorized when access token is not included`() {
-        val rsp = authorizedRequest(
+        val response = authorizedRequest(
             "/begreper/search?orgNummer=123456789",
-            port,
             mapper.writeValueAsString(SearchOperation("test")),
             null,
             HttpMethod.POST
         )
 
-        assertEquals(HttpStatus.UNAUTHORIZED.value(), rsp["status"])
+        assertEquals(HttpStatus.UNAUTHORIZED.value(), response["status"])
     }
 
     @Test
     fun `Forbidden for wrong orgnr`() {
-        val rsp = authorizedRequest(
+        val response = authorizedRequest(
             "/begreper/search?orgNummer=999888777",
-            port, mapper.writeValueAsString(SearchOperation("test")), JwtToken(Access.ORG_READ).toString(),
+            mapper.writeValueAsString(SearchOperation("test")), JwtToken(Access.ORG_READ).toString(),
             HttpMethod.POST
         )
 
-        assertEquals(HttpStatus.FORBIDDEN.value(), rsp["status"])
+        assertEquals(HttpStatus.FORBIDDEN.value(), response["status"])
     }
 
     @Test
     fun `Ok for read access`() {
-        val rsp = authorizedRequest(
+        val response = authorizedRequest(
             "/begreper/search?orgNummer=123456789",
-            port, mapper.writeValueAsString(SearchOperation("test")), JwtToken(Access.ORG_READ).toString(),
+            mapper.writeValueAsString(SearchOperation("test")), JwtToken(Access.ORG_READ).toString(),
             HttpMethod.POST
         )
 
-        assertEquals(HttpStatus.OK.value(), rsp["status"])
+        assertEquals(HttpStatus.OK.value(), response["status"])
+
+        val result: Paginated = mapper.readValue(response["body"] as String)
 
-        val result: Paginated = mapper.readValue(rsp["body"] as String)
         assertEquals(listOf(), result.hits)
     }
 
     @Test
     fun `Query returns correct results`() {
-        val rsp = authorizedRequest(
+        elasticsearchOperations.save(listOf(CurrentConcept(BEGREP_1.toDBO()), CurrentConcept(BEGREP_2.toDBO())))
+        elasticsearchOperations.indexOps(CurrentConcept::class.java).refresh()
```

Comment:
Sikkert vært tjent med å ha dette som en felles metode i feks ContractTestsBase, slik at det i testene blir seende ut som noe ala dette:
```suggestion
        addToElastic(listOf(BEGREP_1, BEGREP_2))
```